starter/package.json

```diff
@@ -143,8 +143,8 @@
   },
   "devDependencies": {
     "@compodoc/compodoc": "~1.1.11",
-    "@nationalbankbelgium/stark-build": "latest",
-    "@nationalbankbelgium/stark-testing": "latest",
+    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.2.0-71462b2f.tgz",
```

Comment:
Shouldn't these changes be reverted? Normally the Starter should install the `latest` from all Stark packages right?